pkg/util/util.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import (
+	"fmt"
+	"github.com/code-ready/crc/pkg/crc/output"
+	"strings"
+)
+
+func PromptUserForYesOrNo(message string) bool {
+	var response string
+	output.OutF(message + "? [Y/N]: ")
+	fmt.Scanf("%s", response)
+	if strings.ToLower(response) == "y" {
+		return true
+	}
+	return false
+}
```

Comment:
I don't want to have util package, in minishift we had so much util part that was very confusing and most of time created our cyclic dep issues :(

Not sure at this point what should we use. @gbraad wdyt?

Reply:
same here... I dislike the idea of a 'util  as we end up putting it in each namespace.

Reply:
it is better to call this `input` as we have an output, right?